components/WeeklyPickButton/WeeklyPickButton.test.tsx

```diff
@@ -3,19 +3,93 @@ import { render, screen } from '@testing-library/react';
 import { WeeklyPickButton } from './WeeklyPickButton';
 import { RadioGroup } from '../RadioGroup/RadioGroup';
 
+// test one method
+// provide some specific arguments to that method
+// test that the result is as expected
+
+const weeklyPickData = {
+  team: 'Ravens',
+  src: '/path/to/ravens.svg',
+  height: '48',
+  width: '48',
+  alt: 'Ravens',
+};
+
 describe('WeeklyPickButton', () => {
-  it('renders correctly', () => {
+  it('RadioGroup has the correct classes', () => {
+    render(<RadioGroup />);
+
+    const radioGroup = screen.getByRole('radiogroup');
+    expect(radioGroup).toHaveClass('grid w-full grid-cols-2 gap-4');
+  });
+
+  it('radio button has the correct classes', () => {
     render(
       <RadioGroup>
-        <WeeklyPickButton team="Test Team" src="/path/to/image.svg"/>
+        <WeeklyPickButton team={weeklyPickData.team} src={weeklyPickData.src} />
```

Comment:
m: Go back into the component and add `data-testid` to the component wrapper and each element you are testing. It will be a bulletproof way to test elements.